phpunit/class-rest-blocks-controller-test.php

```diff
@@ -124,13 +124,16 @@ public function test_create_item() {
 		$response = $this->server->dispatch( $request );
 
 		$this->assertEquals( 200, $response->get_status() );
-		$this->assertEquals(
-			array(
-				'id'      => self::$post_id,
-				'title'   => 'New cool block',
-				'content' => '<!-- wp:core/paragraph --><p>Wow!</p><!-- /wp:core/paragraph -->',
-			), $response->get_data()
-		);
+
+		$data = $response->get_data();
+
+		$this->assertArrayHasKey( 'id', $data );
+		$this->assertArrayHasKey( 'title', $data );
+		$this->assertArrayHasKey( 'content', $data );
+
+		$this->assertEquals( self::$post_id, $data['id'] );
+		$this->assertEquals( 'New cool block', $data['title'] );
+		$this->assertEquals( '<!-- wp:core/paragraph --><p>Wow!</p><!-- /wp:core/paragraph -->', $data['content'] );
```

Comment:
Because this PR adds the `permalink_template` to all post types, it was adding an extra property to the data being returned here. Rather than having to update the tests every time a new property is added, I switched to this method.

Whenever we dump PHP 5.2 support, we can switch to using PHPUnit's `assertArraySubset()` test.